app/utils/geostore.py

```diff
@@ -14,20 +16,21 @@
 async def _get_gfw_geostore_geometry(geostore_id: UUID) -> Geometry:
     """Get GFW Geostore geometry."""
 
-    geo: GeostoreHydrated = await get_geostore_from_anywhere(geostore_id)
-
     try:
-        geometry = geo.gfw_geojson.features[0].geometry
-    except (KeyError, RecordNotFoundError):
+        geo: GeostoreHydrated = await get_geostore_from_anywhere(geostore_id)
+        geometry: Optional[Geometry] = geo.gfw_geojson.features[0].geometry
+    except (KeyError, RecordNotFoundError) as ex:
+        logger.exception(ex)
```

Comment:
logger.execption will always print the traceback. Best to add an explanation what kind of error we caught here to help debugging

Reply:
The question is, do we want to log this as an error, since we catch this and have a response pattern in place. Or should we rather make this a warning?